patches/woocommerce-force-sells.0001.fix.function-called-on-null.patch

```diff
@@ -0,0 +1,83 @@
+From b971735ed630c1e4d1d60db8b6142652d9404636 Mon Sep 17 00:00:00 2001
+From: Andres A <andres@netzstrategen.com>
+Date: Wed, 9 Feb 2022 10:49:24 +0100
+Subject: [PATCH] Fixed function call on null when cart does not exist in session.
```

Comment:
```suggestion
Subject: [PATCH] Fixed fatal error in woocommerce-force-sells if cart instance is created programmatically.
```
Please also rename the keywords in the path filename to give a better idea on what is being fixed:
```diff
-patches/woocommerce-force-sells.0001.fix.function-called-on-null.patch
+patches/woocommerce-force-sells.0001.fix.fatal-error-cart-create-programmatic.patch
```

Reply:
Done.